### DISCUSSION TITLE
Candybar Permissions

### DISCUSSION THREAD

**Discussion by Donnnno** [zixpo/candybar#104] [Q&A]
Hi, I was wondering where all these permissions are, and what function these have within the dashboard:

android.permission.ACCESS_NETWORK_STATE, 
android.permission.ACCESS_WIFI_STATE, 
android.permission.FOREGROUND_SERVICE, 
android.permission.INTERNET, 
android.permission.READ_EXTERNAL_STORAGE, 
android.permission.RECEIVE_BOOT_COMPLETED, 
android.permission.SET_ALARM, 
android.permission.SET_WALLPAPER, 
android.permission.WAKE_LOCK, 
android.permission.WRITE_EXTERNAL_STORAGE, 
android.permission.WRITE_SETTINGS

Some are pretty logical, but others kinda look shady even though they're not used.

Thanks

**Comment by sarsamurmu** [comment]
Some permissions were required by muzei wallpaper. I didn't check if these are still needed with the latest version of muzei.
